config/opensearch_dashboards.yml

```diff
@@ -161,7 +162,38 @@
 #  '2001:db8::/32',
 #  'ff00::/8',
 # ]
-#vis_type_timeline.graphiteBlockedIPs: []
+
+# vis_type_timeline.graphiteBlockedIPs: []
```

Comment:
NIT: for the yaml file we should just show what people should use. We should delete the deprecated and keep the new.

Reply:
fixed!